_posts/2019-10-05-opensource.md

```diff
@@ -93,7 +97,64 @@ To switch from one branch to another use:
 
 <img src= "https://help.github.com/assets/images/help/pull_requests/pullrequest-send.png">
 
-Voila, you just created your first pull request! After this your pull request will reach the organisation and might merge or comment on your changes. 
+Voila, you just created your first pull request! After this your pull request will reach the organisation and might merge or comment on your changes.   
+  
+- To keep your fork updated and in sync with the original, "upstream" repository(from which you have created a fork), we add an upstream and sync our fork by using the following commands:   
+
+```sh
+    $ git remote add upstream <LINK_TO_UPSTREAM_REPOSITORY>
+    $ git fetch upstream
```

Comment:
The sequence of commands should be as follows after adding upstream:
```sh
git checkout master
git fetch upstream
git rebase upstream/master
git push origin master
```

Using `git merge upstream/master` would create a new commit for the merge, which really is unnecessary and only pollutes the git history.